Misc/NEWS.d/next/Library/2020-02-20-01-53-01.bpo-29298.laIn2S.rst

```diff
@@ -0,0 +1 @@
+Fix crash in argparse when using subparsers and Required is set to True.
```

Comment:
```suggestion
Fix crash in :mod:`argparse` when using subparsers and its required attribute
is set to True.
```

Optionally, you can also include a `Contributed by <name>.` sentence at the end of the Misc/NEWS message.